test/set.lean

```diff
@@ -36,3 +36,18 @@ begin
   set b : T := u, -- the type `T` can't be fully elaborated without the body but this is fine
   trivial
 end
+
+section fivefivefive
```

Comment:
`lean_555  -- link-to-the-issue` might be a better name. Or even `\f<<lean#555\f>>`.